diff --git a/src/kyoshin/CGame.hpp b/src/kyoshin/CGame.hpp
index 3e239e4a3..b2e9987e0 100644
--- a/src/kyoshin/CGame.hpp
+++ b/src/kyoshin/CGame.hpp
@@ -17,12 +17,12 @@ public:
     CGame(const char* pName, CWorkThread* pParent);
     virtual ~CGame();
     static CGame* getInstance();
-    static bool func_8003933C();
-    static void func_80039364();
+    static bool isEventReady();
+    static void recreateGame();
     static void setTaskManagerUpdateCount(u32 count);
     virtual void wkUpdate();
     virtual void wkRender();
-    static void func_800395F4(bool r3);
+    static void setLetterbox(bool r3);
     static void setViewRect(CView* view, s16 x, s16 y, s16 width, s16 height);
     virtual bool wkStandbyLogin();
     virtual bool wkStandbyLogout();
